core/src/main/java/org/infinispan/manager/EmbeddedCacheManagerAdmin.java

```diff
@@ -26,4 +48,18 @@
     * @return the cache
     */
    <K, V> Cache<K, V> createCache(String name, Configuration configuration);
```

Comment:
I guess this method will also throw a `CacheException` if the cache already exists (missing in javadoc)